controllers/user/login.js

```diff
@@ -29,6 +34,19 @@ function renderForm(req, res, formValues = null, errors = []) {
     });
 }
 
+function renderRateLimitError(req, res, minutesLeft) {
+    const errorData = {
```

Comment:
Why the extra `errorData` object instead of passing `minutesLeft` straight into the view?